app/plugin.go

```diff
@@ -164,10 +164,17 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 				return nil
 			}
 
-			if fileReader, err := os.Open(walkPath); err != nil {
+			fileReader, err := os.Open(walkPath)
```

Comment:
Any reason for not using `app.FileReader()` here? 
Also we should make sure we close it before returning.